src/pages/edit/index.js

```diff
@@ -249,7 +250,7 @@ const EditProfile = () => {
               </div>
 
               <div class="button-container">
-                <button type="button" class="button offwhite">
+                <button type="button" className="button offwhite">
                   Cancel
                 </button>
                 <button type="button" class="button blue">
```

Comment:
this should also have a className prop, no?

Reply:
It should, will update it now